src/Components/hero.html

```diff
@@ -0,0 +1,23 @@
+<section id="main " class="grid grid-cols-2 mt-[50px]">
+
+    <div class="text-white text-[60px] font-ralewayBold">
+
+<p>Win-Win:</p>
+<p class="flex gap-5"> <span>Europas</span> <span  class="text-primary">Digitalisierung </span></p>
+<p>profitiert von Afrikas Talenten</p>
+
+  
+
+
+</div>
+
+
+<div class="overflow-x-hidden -ml-10 translate-x-10 translate-y-11 scale-125">
+    <img class="" src="/src/Assets/media/Ojembees1.webp" alt="picture of ojemba developers">
+</div>
+
+
+
+
+
```

Comment:
The photo can be overflow to the right as in the figma  design

![here kev](https://github.com/Bateyjosue/ojemba-team-project/assets/52311487/ced69d34-9b64-4997-8716-fd79f04ae49e)
![Hero design](https://github.com/Bateyjosue/ojemba-team-project/assets/52311487/865e75c1-74f5-4aa6-bf6a-9478b34d47a2)